Caffe_Americano.rs

```diff
@@ -1,6 +1,12 @@
 /// 0,2l Kaffee aus Espressobohnen
 /// Stark, mächtig, sicher,
 /// eine echte Alternative zu C++
+
+fn name_with_price(price: &str) -> String {
```

Comment:
if you really wanna show of the languages, how about a more strongly typed `Price` and `Product`?

Reply:
Oh nice, instead of just having a function returning stuff everywhere do something a little more idiomatic, I like that.